### ISSUE TITLE
Can't choose a size of uploaded picture on editing post in Wordpress

### ISSUE THREAD

**Issue by alexjc347** [OpenLiveWriter/OpenLiveWriter#605]
I have a Wordpress self-hosted site. After send new post by Open Live Writer to blog I try to edit it.
But in edit window of picture, there are don't exist a list of thumb sizes.
Also OLW processing all pictures and they after have more kbytes.
Sorry for my english.
http://prntscr.com/ev50ap

**Comment by ScottIsAFool** [comment]
So in OLW, if you click on the image in your post, you don't see this when you click on the Format tab?
![image](https://cloud.githubusercontent.com/assets/1658896/24912886/62c221c2-1ec7-11e7-9ea2-f00b04949ecd.png)

**Comment by alexjc347** [comment]
I see this
http://prntscr.com/ev68ka

**Comment by ScottIsAFool** [comment]
Yes, so if you look just under where the Insert tab is, you can see it says "Small", that's where you would change the size of the image.

**Comment by ScottIsAFool** [comment]
Live Writer has its own options, it isn't based on the blogging platform.

**Comment by alexjc347** [comment]
If you upload the image as well as through the WP loader, then everything should be fine

**Comment by ScottIsAFool** [comment]
Is this when editing the post in Wordpress? If so, Writer shouldn't be affecting this.

**Comment by alexjc347** [comment]
But if you upload a picture through the WP all is normal. And if through the OWL, the WP does not understand that the picture has several sizes, although the files in the folder wp-contents\uploads are created for each size. It is very strange.

**Comment by ScottIsAFool** [comment]
Ok, that's probably WP behaviour where it will create the different thumbnail sizes when you upload the picture, Live Writer doesn't do this.

**Comment by alexjc347** [comment]
I think it is relate, that as on the image from OWL it is impossible to select a link to the mediafile and to the attachment page as here
http://prntscr.com/ev6g6y

**Comment by alexjc347** [comment]
This is strange, but WP creates all three thumbnails, but does not show a list of them

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
